acp/ideas_module.php

```diff
@@ -42,12 +45,18 @@ class ideas_module
 	/** @var string */
 	public $u_action;
 
+	/**
+	 * Constructor
+	 *
+	 * @access public
+	 */
 	public function __construct()
 	{
-		global $config, $db, $phpbb_log, $request, $template, $user, $phpbb_root_path, $phpEx;
+		global $config, $db, $phpbb_container, $phpbb_log, $request, $template, $user, $phpbb_root_path, $phpEx;
 
 		$this->config = $config;
 		$this->db = $db;
+		$this->language = $phpbb_container->get('language');;
```

Comment:
Well, it consumes more memory to use the container to get every service, which is not needed since they already exist in the global scope. The solution is for the ACP to move forward and use services like the rest of phpBB, but that's not happening in 3.2.